language-support/scala/bindings-akka/src/test/scala/com/digitalasset/ledger/client/binding/retrying/CommandRetryFlowUT.scala

```diff
@@ -34,7 +34,7 @@ class CommandRetryFlowUT extends AsyncWordSpec with Matchers with AkkaTest {
     Flow[In[RetryInfo[Status]]]
       .map {
         case Ctx(context @ RetryInfo(_, _, _, status), SubmitRequest(Some(commands), tc)) =>
-          if (commands.ledgerEffectiveTime.get.seconds == 0) {
+          if (commands.deduplicationTime.get.nanos == 0) {
```

Comment:
We should probably check `seconds`. `nanos` will be empty if you simply select `30.seconds`.

Reply:
As far as I understood this test, it used the LET as a kind of index (retry number `i` had `LET=i`). I have switched to using `deduplicationTime` instead, and to using nanos instead of seconds in order to not create large timeouts.

The deduplication time is initialized with `com.google.protobuf.duration.Duration.of(120, 0)`